### PR TITLE
Update W2RomanType.cpp

### PR DESCRIPTION
**Pull request by einstein2009**

### CODE REVIEW

File: RomanNumerals/W2RomanType.cpp [hunk 1]
```diff
@@ -16,36 +17,38 @@ using namespace std;
 
 void romanType::setRomanNum(char rNum)
 {
-	strncpy (romanNum, rNum, sizeof(rNum));
+// need to figure out.
```
Comment (EthanEChristian) [inline RomanNumerals/W2RomanType.cpp]: maybe a for loop?

File: RomanNumerals/W2RomanType.cpp [hunk 2]
```diff
@@ -16,36 +17,38 @@ using namespace std;
...
 }
 
 char romanType::getRomanNum()
 {
-	return for (int i = 0; i <= 10; i++){romanNum[10]};
+	for (int i = 0; i <= 10; i++)
+	{
+	return romanNum[10];
```
Comment (EthanEChristian) [inline RomanNumerals/W2RomanType.cpp]: Boo! A method only returns once

File: RomanNumerals/W2RomanType.cpp [hunk 3]
```diff
@@ -16,36 +17,38 @@ using namespace std;
...
-		else if (romanNum[i] = 'L')
-			posInt += 50;
-		else if (romanNum[i] = 'X')
-			posInt += 10;
-		else if (romanNum[i] = 'V')
-			posInt += 5;
-		else if (romanNum[i] = 'I')
-			posInt += 1;
+	{
+		if (roman[rNum[i]] < roman[rNum[i]])
```
Comment (EthanEChristian) [inline RomanNumerals/W2RomanType.cpp]: always false because it always equals

File: RomanNumerals/W2RomanType.cpp [hunk 4]
```diff
@@ -16,36 +17,38 @@ using namespace std;
...
+			posInt -= romanrNum[i]];
 		else
-			cout << "Invalid entry: " << romanNum[i] << endl;
+			posInt += roman[rNum[i]];
 	}
-
+	posInt += roman[rNum[i] -1];
```
Comment (EthanEChristian) [inline RomanNumerals/W2RomanType.cpp]: huh?

File: RomanNumerals/W2RomanType.cpp [hunk 5]
```diff
@@ -68,9 +71,9 @@ bool romanType::pickInt()
 		cout << endl;
 	
 		if (format == 'I' || format == 'i')
-			return 1;		//true
+			return true;
```
Comment (EthanEChristian) [inline RomanNumerals/W2RomanType.cpp]: +1

File: RomanNumerals/W2RomanType.cpp [hunk 6]
```diff
@@ -68,9 +71,9 @@ bool romanType::pickInt()
...
 		else
-			return 0;		//false
+			return false;
```
Comment (EthanEChristian) [inline RomanNumerals/W2RomanType.cpp]: +1

File: RomanNumerals/W2RomanType.cpp (hunk 3 above)
Comment (einstein2009) [inline RomanNumerals/W2RomanType.cpp]: if (roman[rNum[i]] < roman[rNum[i + 1]])

**Pull request merged**